### DISCUSSION TITLE
Feature Request: Support privileged containers

### DISCUSSION THREAD

**Discussion by andyndang** [cloudnative-pg/cloudnative-pg#1934] [Ideas]
We need to run additional code with privileged permission (touching the file system mount logic) and thus would like to have this as an option in the cluster spec.

**Comment by sxd** [comment]
Hi @andyndang 

Which fil system you want to modify? if the case it's on the pod file system, why you will need to do it? The design it's about not doing that kind of thing to keep the immutability of the pods and containers, also, for security reasons that shouldn't be allow unless the app it's designed like that, in this case, PostgreSQL it's allow to create/delete/modify files in it's own mount path, and the rest of the file system it's kept immutable.

Please, provide more feedback about why you need this, thank you!

I'm turning this into a discussion, because isn't an issue, yet, if turn out to be one, we create a ticket from there.

Regards!
